src/tracer/build.rs

```diff
@@ -1,4 +1,4 @@
 fn main() {
     // useful to trigger changes in the cargo build on the json file
-    println!("cargo:rerun-if-changed=src/common/target_process/default_rules.json");
+    println!("cargo:rerun-if-changed=src/common/target_process/yaml_rules/tracer.rules.yml");
```

Comment:
The directory is `yml_rules` and you probably want to rerun if any of the files in that folder are changed